CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Release Notes
 -------------
 
+**1.12.1 (2016-12-19)**
```

Comment:
I don't know if pytest-html follows semantic versioning, but if it does, shouldn't this be `1.13.0`? I think it makes sense to make new minor release so people can pin to it instead of pinning to a patch release.